client/my-sites/google-my-business/new-account/index.js

```diff
@@ -53,11 +54,13 @@ class GoogleMyBusinessNewAccount extends Component {
 					title="Google My Business > New"
 				/>
 
+				<DocumentHead title={ translate( 'Google My Business' ) } />
+
 				<HeaderCake isCompact={ false } alwaysShowActionText={ false } onClick={ this.goBack }>
 					{ translate( 'Google My Business' ) }
 				</HeaderCake>
 
-				<Card className="gmb-new-account__card">
+				<Card>
```

Comment:
We need a wapper div inside this which adds a `max-width: 660px;` rule.

Reply:
I'm not sure to understand. Why would we want to reduce the width of this specific card? All other Google My Business pages and cards have a width of 992 pixels.

Reply:
We don't need to reduce the width of the card, we need to reduce the width of the text inside the card, so that the line length doesn't get too long (see how's its done in #22801 )